assembl/models/notification.py

```diff
@@ -713,10 +724,14 @@ class ModelEventWatcherNotificationSubscriptionDispatcher(BaseModelEventWatcher)
     protocol"""
 
     def processPostCreated(self, objectId):
-        self.createNotifications(objectId)
+        self.createNotifications(objectId, None)
+
+    def processPostModified(self, objectId, state_changed):
+        if state_changed:
+            self.createNotifications(objectId, state_changed)
 
     @classmethod
-    def createNotifications(cls, objectId):
+    def createNotifications(cls, objectId, state_changed=None):
```

Comment:
You added `state_changed` to createNotifications method, but you don't seem to use it.

Reply:
Good point, I intended to use it but ended up not doing it.